index.bs

```diff
@@ -1511,8 +1511,8 @@ and an optional [=attribution report=] <dfn for="obtain debug data on trigger re
 <h3 algorithm id="triggering-event-level-attribution">Triggering event-level attribution</h3>
 
 To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |trigger|, an
-[=attribution source=] |sourceToAttribute|, an [=attribution rate-limit record=] |rateLimitRecord|
-and a [=boolean=] |topLevelFiltersMatch|, run the following steps:
+[=attribution source=] |sourceToAttribute|, and an
+[=attribution rate-limit record=] |rateLimitRecord|, run the following steps:
 
 1. If |trigger|'s [=attribution trigger/event-level trigger configurations=]
```

Comment:
I'm going to be reorganizing this in a followup, specifically to avoid sending any verbose debug reports if both the event trigger data list is empty and the aggregatable trigger data list is empty. Otherwise, verbose debug reports should be sent if at least one of them would have proceeded.

Reply:
SGTM.